server/lobbyconnection.py

```diff
@@ -36,6 +37,34 @@
 from .protocol import QDataStreamProtocol
 from .rating import RatingType
 from .types import Address
+from  .clientmessages import *
+from .docstrings import trim_docstring
+
+
+__pdoc__ = {}
+COMMAND_HANDLERS = {}
+
+def handler(handling_class):
+    def decorator(function: Callable) -> Callable:
+        COMMAND_HANDLERS[handling_class] = function
+
+        func_string = function.__doc__ or "[Method description missing]"
```

Comment:
This is also visually unpleasing. Maybe adding italics could make it more clear.

```suggestion
        func_string = function.__doc__ or "*[Method description missing]*"
```